starter-code/views/about.hbs

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html class="aboutme" lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <script type="module" src="./js/client-scripts.js"></script>
+</head>
+<title>Foody ♥</title>
+
+<body>
+    {{>nav}}
+
+    <h2>About</h2>
+    <div class="about">
+        <img class="girl" src="/images/about.jpg" alt="aphoto" style="width:20%;height:20%;">
```

Comment:
It's cool that you're using alt tags, but adding a meaningful text is important - it helps for people using screen readers